configure

```diff
@@ -260,14 +260,15 @@ opt_core() {
 
     if [ $HELP -eq 0 ]
     then
+        local UOP=$(echo $OP | tr '[:lower:]-' '[:upper:]_')
+        local UFLAG=$(echo $FLAG | tr '[:lower:]' '[:upper:]')
+        local V="CFG_${UFLAG}_${UOP}"
+        eval $V=0
         for arg in $CFG_CONFIGURE_ARGS
         do
             if [ "$arg" = "--${FLAG}-${OP}" ]
             then
-                OP=$(echo $OP | tr 'a-z-' 'A-Z_')
-                FLAG=$(echo $FLAG | tr 'a-z' 'A-Z')
-                local V="CFG_${FLAG}_${OP}"
-                local V_PROVIDED="CFG_${FLAG}_${OP}_PROVIDED"
```

Comment:
Could you elaborate a bit on what's going on here? This seems unrelated to the makefile changes and our configure script is historically *very* tricky to modify